var/spack/repos/builtin/packages/mpas-model/package.py

```diff
@@ -69,6 +69,9 @@ class MpasModel(MakefilePackage):
     depends_on("mpi")
     depends_on("parallelio")
 
+    # `ifx` internal compiler error triggered by maps-model fixed in oneapi@2024.2
+    conflicts("%oneapi@:2024.1")
```

Comment:
Can you turn the comment into a short message?

Reply:
Done.